features/claims/advocate/scheme_eleven/advocate_supplementary_claim_submit.feature

```diff
@@ -44,16 +44,16 @@ Feature: Advocate tries to submit a supplementary claim for miscellaneous fees (
       | miscellaneous | Wasted preparation fee |
 
     When I select an advocate category of 'Junior'
-    And I choose the 'Confiscation hearings (half day uplift)' miscellaneous fee with quantity of '1'
     And I choose the 'Confiscation hearings (half day)' miscellaneous fee with quantity of '2'
+    And I choose the 'Confiscation hearings (half day uplift)' miscellaneous fee with quantity of '1'
```

Comment:
Does the ordering of these matter?
* If no, then do the steps need to be changed?
* If yes, then do both cases need to be tested?

Reply:
no they don't, as mentioned in the commit they are merely to reflect the expected ordering

Reply:
yes they do need testing. this was the only test at the time that caught the problem and is a general blackbox feature test